docs/pages/2020_Intro_Week/sections/workflows_adv.rst

```diff
@@ -200,7 +216,7 @@ We can now launch it like any other work chain and the ``BaseRestartWorkChain``
 
     submit(ArithmeticAddBaseWorkChain, x=Int(3), y=Int(4), code=load_code('add@localhost'))
 
-Once the work chain finished, we can inspect what has happened with, for example, ``verdi process status``:
+Once the work chain finished, we can inspect what has happened with, for example, ``verdi process status PK``, where ``PK`` is the ``pk`` of the submitted work chain process:
```

Comment:
I wouldn't explicitly state ``PK`` because it also accepts UUIDs. Anyway, if they don't know this by now in the "advanced" workflow section, I don't know what to say